GTG/gtk/editor/editor.py

```diff
@@ -60,30 +65,30 @@ def __init__(self,
         self.clipboard = clipboard
         self.builder = Gtk.Builder()
         self.builder.add_from_file(GnomeConfig.EDITOR_UI_FILE)
-        self.donebutton = self.builder.get_object("mark_as_done_editor")
-        self.dismissbutton = self.builder.get_object("dismiss_editor")
-        self.deletebutton = self.builder.get_object("delete_editor")
+        self.donebutton = self.builder.get_object("mark_as_done")
+        self.dismissbutton = self.builder.get_object("dismiss")
+        self.deletebutton = self.builder.get_object("delete")
         self.deletebutton.set_tooltip_text(GnomeConfig.DELETE_TOOLTIP)
         self.subtask_button = self.builder.get_object("insert_subtask")
-        self.subtask_button.set_tooltip_text(GnomeConfig.SUBTASK_TOOLTIP)
-        self.inserttag_button = self.builder.get_object("inserttag")
-        self.inserttag_button.set_tooltip_text(GnomeConfig.TAG_TOOLTIP)
-        self.open_parents_button = self.builder.get_object("open_parents")
-        self.open_parents_button.set_tooltip_text(
-            GnomeConfig.OPEN_PARENT_TOOLTIP)
+        # self.subtask_button.set_tooltip_text(GnomeConfig.SUBTASK_TOOLTIP)
+        self.inserttag_button = self.builder.get_object("tags")
```

Comment:
as you set the tooltip in .ui file, you don't need the reference to this object any more. Drop the line! :fire: